.yarn/patches/@metamask-transaction-controller-npm-56.1.0-b106223e34.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/dist/utils/batch.cjs b/dist/utils/batch.cjs
```

Comment:
Patched by @vinistevam and was originally introduced by https://github.com/MetaMask/metamask-extension/pull/33349

Reply:
I think I found the correct one: https://github.com/MetaMask/metamask-extension/pull/33269